lib/enkf/tests/enkf_workflow_job_test_directory.cpp

```diff
@@ -21,58 +21,34 @@
 #include <ert/util/test_util.h>
 #include <ert/enkf/ert_test_context.hpp>
 
-/*
-  These version functions are overrideed here to get a consistent
-  version mapping.
-*/
 
-int ecl_version_get_major_version( ) {
-  return 1;
-}
-
-
-int ecl_version_get_minor_version( ) {
-  return 2;
-}
-
-
-const char* ecl_version_get_micro_version( ) {
-  return "3";
-}
-
-
-void test_version() {
-  test_assert_int_equal( ecl_version_get_major_version( ) , 1 );
-  test_assert_int_equal( ecl_version_get_minor_version( ) , 2 );
-  test_assert_string_equal( ecl_version_get_micro_version( ) , "3" );
-}
 
 
 int main(int argc , const char ** argv) {
   enkf_main_install_SIGNALS();
-  test_version( );
   {
     const char * path = argv[1];
     ert_workflow_list_type * workflows = ert_workflow_list_alloc_empty(NULL);
     ert_workflow_list_add_jobs_in_directory( workflows , path );
 
+
     // The CONF1 only exists as default - unversioned
     test_assert_true( ert_workflow_list_has_job( workflows , "CONF1"));
 
     // The CONF2 exists as the default - which is invalid and will not load,
```

Comment:
guess since the assertions is swapped the comments are outdated and should be changed / removed as well?